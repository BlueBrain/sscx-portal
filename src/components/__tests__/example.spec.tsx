import React from 'react';
import { shallow } from 'enzyme';
import Filter from '../example';

describe('', () => {
  it('filter renders correctly', () => {
    const result = shallow(<Filter />);
    expect(result).toMatchSnapshot();
  });
});
