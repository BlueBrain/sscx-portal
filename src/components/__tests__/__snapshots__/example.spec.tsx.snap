// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter renders correctly 1`] = `
<h1>
  SSCx portal
</h1>
`;
